code/obj/machinery/door/airlock.dm

```diff
@@ -1692,12 +1698,19 @@ About the new airlock wires panel:
 
 	playsound(src.loc, src.sound_airlock, 25, 1)
 
-	if (src.cycle_id)
-		for (var/obj/machinery/door/airlock/D in cycling_airlocks[src.cycle_id])
+	if (src.junction_id)
```

Comment:
Is there a good way to skip this or otherwise handle tug-of-wars from e.g. Kudzu opening doors? See #18024

Reply:
tiny bit out of scope, but i'm planning another PR that will give them safety light integration. I'd like to just keep this PR as mainly packets based for now.

The plan for that is to have safety light being off 'disabling' the cycling integration, so that they don't get shut by others and also don't send a signal to shut others. And turning the light back on re enables the behaviour (it doesn't disconnect it from the junction, just disables that behaviour temporarily). The idea being that kudzu could then disable the safety/power on doors it takes over, via its vines being physically inside the door or sth.